backend/b2/b2.go

```diff
@@ -696,9 +703,20 @@ func (f *Fs) list(ctx context.Context, bucket, directory, prefix string, addBuck
 		Method: "POST",
 		Path:   "/b2_list_file_names",
 	}
-	if hidden {
+	if hidden || versionAt != "" {
 		opts.Path = "/b2_list_file_versions"
 	}
+
+	var versionAtTime time.Time
```

Comment:
This should be parsed into a time.Time in NewFs so we only do it once at the start.

We can then use `f.versionAtTime.IsZero()` to see if the feature is active.